measurements/geography_based.py

```diff
@@ -64,29 +64,46 @@ def geo_ctr(dataloader, args):
 def geo_ctr_gps(dataloader, args):
     # import custom political boundaries shapefile from dataset 
     geo_boundaries = dataloader.dataset.geo_boundaries
+    geo_boundaries_key_name = dataloader.dataset.geo_boundaries_key_name
+
+    # import subregion boundaries shapefile
+    subregion_boundaries = dataloader.dataset.subregion_boundaries
```

Comment:
this is only relevant if the dataset is "global" right? else this is not necessary. perhaps we can define "global" for our use-case to be containing at least two distinct subregions

Reply:
changed convention to be self.geography_info_type = "COUNTRY_LABEL" # GPS_LABEL or COUNTRY_LABEL or REGION_LABEL

Regarding distinction between country and region label, I found that abstracting away this from the user proved to be pretty tricky. Will follow up with you on this